angular-rails-app/lib/tasks/db_connector.rake

```diff
@@ -54,12 +54,12 @@ namespace :db_connector do
     sql = "INSERT IGNORE INTO r_properties "\
         "(street_address,city,state,zipcode,hpd_registration_id,borough,block,lot,bbl,created_at,updated_at) "\
         "SELECT CONCAT(TRIM(housenumber), ' ', TRIM(streetname)), boro, 'New York', zip, registrationid, boro, block, lot, bbl, NOW(),NOW() "\
-        "FROM hpd_buildings WHERE streetname != '' AND streetname IS NOT NULL AND registrationid != 0;"
+        "FROM #{ENV['MYSQL_DATABASE_DATA']}.hpd_buildings WHERE streetname != '' AND streetname IS NOT NULL AND registrationid != 0;"
     conn.execute(sql)
 
     # Add unit counts to HPD data.
     sql = "UPDATE r_properties "\
-        "INNER JOIN pluto_nyc ON r_properties.bbl = pluto_nyc.bbl AND r_properties.total_units IS NULL "\
+        "INNER JOIN #{ENV['MYSQL_DATABASE_DATA']}.pluto_nyc ON r_properties.bbl = #{ENV['MYSQL_DATABASE_DATA']}.pluto_nyc.bbl AND r_properties.total_units IS NULL "\
```

Comment:
Nope, the r_ tables are default, since the database is specified as part of the connection.

Speaking of r_, I've also removed the ActiveRecord config to prefix.